data/json/recipes/food/carnivore.json

```diff
@@ -0,0 +1,72 @@
+[
```

Comment:
Pizza and pie items were ported over and originally required foil, before being dropped as not sensible.